query/schema/Orders.js

```diff
@@ -0,0 +1,31 @@
+cube(`Orders`, {
```

Comment:
how do we create new schemas for newer datasets?

Reply:
Working on that

Reply:
Working on that